gridt/src/app/movements/movements.page.html

```diff
@@ -52,19 +52,19 @@
             <!-- Right now, this just shows every single movement -->
             <!-- Display the title on the left -->
             <ion-label text-wrap>
-              <h3>{{ movement.name }}</h3>
-              <p>{{ movement.short_description }}</p> 
+              <h3>{{ movement.name }} <ion-icon *ngIf="movement.subscribed" name="people-sharp"></ion-icon> </h3>
```

Comment:
I feel like the indenting and organizing of the code could be a bit clearer here.